src/Core/Repositories/SqlServer/OrganizationUserRepository.cs

```diff
@@ -391,5 +391,18 @@ public async Task ReplaceManyAsync(IEnumerable<OrganizationUser> organizationUse
                 return results.ToList();
             }
         }
+
+        public async Task<IEnumerable<OrganizationUserUserDetails>> GetManyByMinimumRoleAsync(Guid organizationId, OrganizationUserType baseRole)
+        {
+            using (var connection = new SqlConnection(ConnectionString))
+            {
+                var results = await connection.QueryAsync<OrganizationUserUserDetails>(
+                    "[dbo].[OrganizationUser_ReadByRole]",
```

Comment:
You also forgot to update the SPROC name here....

Reply:
Fixed